truenas/webui
# NAS-117576 / 22.12 / NAS-117576: Optimizing calls on storage dashboard

Some calls were optimized, but you may still face an error if you have a lot of pools. A hack was introduced to make it less likely.

Testing:
* Check loading on storage dashboard for many pools.
* Check usage card.
